pages/index.mdx

```diff
@@ -1,3 +1,30 @@
 # Digital Garden 🌱
 
-Welcome to my Digital Garden. I will be adding some pages here about a bunch of things I find interesing.
+Welcome to my Digital Garden. This is an extension of [my portfolio](https://ineza.codes).
+The idea of a digital garden came after realising that I had no place to store useful information.
+
+I wanted a website where I could;
+
+- Write my thoughts or discoveries that are related to my coding journey.
+- Save useful links/bookmarks that I (or someone else) might require at a later time
+- Save useful tools I've come across in case I ever forget them
+- Write documentation about the projects I've worked on.
+
+My first step was opening [issue #122](https://github.com/inezabonte/inezabonte/issues/122) on my
+portfolio's repo as I began researching on how I would make it possible.
+
+My aim was to worry less about the look of the site but more about the content.
+I first came across [Storybook](https://storybook.js.org/). I had no idea what it was used for,
+I later came to learn it's used mostly to develop design systems and building components in an isolated
+environment. Storybook is nice but, it's a bit too complex for what I was looking for.
+
+After searching I came across [Nextra](https://nextra.vercel.app/) and [docz](https://www.docz.site/).
+They are both static site generators that use [MDX](https://mdxjs.com/) for the content which is a plus.
+The difference between the two is that Nextra is powered by [Next.js](https://nextjs.org/) while docz is powered by [Gatsby](https://www.gatsbyjs.com/).
+Docz having been around for a longer time has the advantage of being well known with more users and more features as well.
+I however chose Nextra because I have been building with Next.js for a while and wasn't willing to
+let it go just yet and also because of its simple look and easiness to setup.
+
+Anyway, feel free to have a look around. Hope it inspires you to make something of your own 😊
+
+You can find me on [Twitter](https://twitter.com/inezabonte) if you would like to connect :)
```

Comment:
```suggestion
You can find me on [Twitter](https://twitter.com/inezabonte) if you like to connect :)
```
I don't think you should use a [softener](https://happyrubin.com/nlp/language-softeners/) here.